security/pkg/nodeagent/caclient/providers/vault/client.go

```diff
@@ -19,70 +19,161 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"fmt"
+	"io/ioutil"
 	"net/http"
 	"strconv"
+	"strings"
 
-	"github.com/hashicorp/vault/api"
+	vaultapi "github.com/hashicorp/vault/api"
 
-	caClientInterface "istio.io/istio/security/pkg/nodeagent/caclient/interface"
+	"istio.io/istio/security/pkg/util"
+	"istio.io/pkg/env"
 	"istio.io/pkg/log"
 )
 
+const (
+	envVaultAddr        = "VAULT_ADDR"
```

Comment:
I find this style very hard to read and inconsistent with rest of Istio. 

Pilot has the env name in-line - (
```	MaxConcurrentStreams = env.RegisterIntVar(
		"ISTIO_GPRC_MAXSTREAMS",
		100000,
		"Sets the maximum number of concurrent grpc streams.",
	).Get()
```

For security code you always need to navigate to find the actual name - and because the comments and defaults are missing on the name, it's pretty hard to comprehend.